module/Makefile

```diff
@@ -0,0 +1,26 @@
+# find the OS
+uname_S := $(shell sh -c 'uname -s 2>/dev/null || echo not')
```

Comment:
We should not be spending any effort to support macOS (or any OS that isn't the particular version of Ubuntu in use on the linux.cs servers and on the VM). Please remove all non-Ubuntu support from this `Makefile`.